src/components/organisms/dashboard/welcomeMainBlock.tsx

```diff
@@ -34,7 +34,7 @@ export const DashboardWelcomeMainBlock = () => {
 		<div className="z-10 mt-7 grid select-none gap-5">
 			<div className="col-span-1 grid grid-cols-auto-fit-350 items-stretch gap-4 overflow-visible rounded-2xl border border-gray-950 bg-black p-8 pl-6 pr-4 font-averta text-white">
 				<div className="flex min-h-52 flex-col">
-					<div className="flex w-full flex-1 items-center justify-center rounded-2xl border-2 border-gray-750 bg-[#1d2226] bg-[url('/assets/image/pages/intro/main.jpg')] bg-contain bg-center bg-no-repeat">
+					<div className="flex w-full flex-1 items-center justify-center rounded-2xl border-2 border-gray-750 bg-[#1d2226] bg-[url('@assets/image/pages/intro/main.jpg')] bg-contain bg-center bg-no-repeat">
```

Comment:
so, also add `@`  for IntroMainBlock component

Reply:
refactored